### ISSUE TITLE
trunc bif fails most testcases

### ISSUE THREAD

**Issue by rvjansen** [adesutherland/CREXX#204]
ts_trunc.rexx

**Comment by Peter-Jacob** [comment]
I corrected one float to string conversion error.
But I don't understand these tests:

```
if trunc(10000000.05,2) \= 10000000.10 then say 'failed in test 16 '
...
if trunc(10000000.055,2) \= 10000000.10 then say 'failed in test 20 '
...
```
TRUNC does not round per definition:
Regina manual
```
Note 
that number is never rounded, except for the rounding that might take place during normalization.
```

I tested also in BREXX, they fail too, there is also no rounding there.

**Comment by adesutherland** [comment]
I think (although the ANSI standard does not seem 100% clear to me) that what should be happening is that the 10000000.05 is first being rounded (in fact converted from a string to a number) as per the numeric setting (of 9) and then it is truncated.

This behaviour is never going to work for cREXX level b - we will have to do it when we have the arbitrary numbers class for level c

**Comment by rvjansen** [comment]
ooRexx clears all the test cases.
But I understand why crexx level b will not, and I'll remove this from the running testcases until we have decimal.

**Comment by rvjansen** [comment]
as does VM Rexx. but case closed.

**Comment by Peter-Jacob** [comment]
some errors corrected (beside the roundings): commit 0e0254fe71c9990a4c4a27edca5ff242a9eac209
